tests/test_setup_config_uncovered.py

```
@@ -0,0 +1,2280 @@
+@patch("fetchtastic.setup_config.setup_reboot_cron_job")
+@patch("fetchtastic.setup_config.install_termux_packages")
+@patch("fetchtastic.setup_config.check_storage_setup")
+@patch(
+    "fetchtastic.setup_config.get_fetchtastic_installation_method",
+    return_value="pipx",
+)
+@patch("fetchtastic.cli.main")
+@patch("shutil.which")
+@patch(
+    "fetchtastic.setup_config.platformdirs.user_config_dir",
+    return_value="/tmp/config",
```

Ruff (0.15.7)

[error] Probable insecure usage of temporary file or directory: "/tmp/config"

(S108)